licenses.yaml

```diff
@@ -5006,6 +5006,23 @@ libraries:
   - com.sun.activation: jakarta.activation
 
 ---
```

Comment:
We can also [remove](https://github.com/apache/druid/blob/master/licenses.yaml#L3747) the [license](https://github.com/apache/druid/blob/master/licenses.yaml#L3735) and [label](https://github.com/apache/druid/blob/master/LABELS#L61) entries for `jaxb-api`.